xonsh/parsers/base.py

```diff
@@ -238,20 +237,6 @@ def raise_parse_error(
     raise err
 
 
-class YaccLoader(Thread):
-    """Thread to load (but not shave) the yacc parser."""
-
-    def __init__(self, parser, yacc_kwargs, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.daemon = True
-        self.parser = parser
-        self.yacc_kwargs = yacc_kwargs
-        self.start()
-
-    def run(self):
-        self.parser.parser = yacc.yacc(**self.yacc_kwargs)
-
-
```

Comment:
Does this mean, the parser is loaded eagerly? I have all my config in pure Python and parser is not needed for the startup making it faster with threaded version